src/utils/pipes/json.js

```diff
@@ -1,9 +1,18 @@
-export class JSONPipe extends Pipe {
- supports(obj) {
-   return true;
- }
+import { Pipe } from 'angular2/change_detection';
 
- transform(value) {
-   return JSON.stringify(value, null, 2);
- }
+export class JSONPipeFactory extends Pipe {
+  constructor() {
+    super()
+  }
+  supports(obj):boolean {
+    return true;
+  }
+
+  transform(value):string {
+    return JSON.stringify(value, null, 2);
+  }
+
+  create(cdRef):Pipe {
+    return new JSONPipeFactory(cdRef);
```

Comment:
can't we just return this :)? We're not using the constructor anyway and we avoid creating another object